SteamKit2/SteamKit2/Steam/SteamClient/CallbackMgr/CallbackMgr.cs

```diff
@@ -112,7 +111,9 @@ public IDisposable Subscribe<TCallback>( JobID jobID, Action<TCallback> callback
                 throw new ArgumentNullException( nameof(callbackFunc) );
             }
 
+#pragma warning disable CA2000 // Not implicitly disposed
```

Comment:
I suspect this is warning because the `Subscription` constructor could throw, and this remains undisposed.